near/omni-types/src/lib.rs

```diff
@@ -393,6 +393,7 @@ pub struct FastFinTransferMsg {
     pub fee: Fee,
     pub msg: String,
     pub storage_deposit_amount: Option<u128>,
+    pub relayer: AccountId,
```

Comment:
Perhaps we can make it optional and take sender_id otherwise?

Reply:
Yeah, I considered doing that too, but I realized that the risk of failing to pass this is greater than the inconvenience of adding it.